docker-vertica/Dockerfile

```diff
@@ -73,7 +74,6 @@ RUN set -x \
   && chmod a+x /opt/vertica/bin/docker-entrypoint.sh \
```

Comment:
We should take this opportunity to cleanup the builder image a bit.  A couple things we should do: 
- remove the centos8 conditionals.  We seem to be firmly entrenched with centos7.  
- remove `yum -y update --security` and the comment that goes with it.  Its weird to get the latest security fixes for the builder image because we throw it all away.